tcks/pom.xml

```diff
@@ -36,4 +36,16 @@
         </plugins>
     </build>
 
+    <dependencyManagement>
```

Comment:
Would it be better if we excluded any arquillian dependencies from the TCK dependencies instead of doing it this way?

Reply:
I haven't tried that to be honest, but when looking for build failure details in the log I think I noticed that the TCKs here were using more of Arquillian than what was clashing with the rest of the project, so completely excluding arquillian might not work and I thought relying on depenencyManagement would have been simpler / less fragile.

Reply:
Ok, but the version of Arquillian is also present in the test framework module so it would be good if we defined a property somewhere for it

Reply:
OK, just added in build-parent/pom.xml which is what the tcks and test-framework have in common.